lib/metrics/processRequests.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var Gauge = require('../gauge');
+
+module.exports = function() {
+    var gauge = new Gauge('node_active_handles_total', 'Number of active requests.');
+
+    return function() {
+        gauge.set(process._getActiveRequests().length);
```

Comment:
_getActiveRequests and _getActiveHandles are new stuff to me, tried googling it and didn't find much info in node documentation. Is it a good practice to depend on "private" functions?